yaksh/models.py

```diff
@@ -124,7 +127,10 @@ def dict_to_yaml(dictionary):
 
 
 def get_file_dir(instance, filename):
-    upload_dir = instance.lesson.name.replace(" ", "_")
+    if isinstance(instance, LessonFile):
```

Comment:
Just suggestion.
In future I feel will be better if we have a separate function, for each instances.
Eg: get_lesson_dir, get_video_dir ...
This will reduce the ifs in a single function.
Also if a new person is implementing similar thing then may write his own callable as per the need and no need to worry about the use of the existing function.

Reply:
No issue here.
Just that two different instances were seen in the same function, thought of mentioning.